activemodel/lib/active_model.rb

```diff
@@ -52,7 +53,9 @@ module ActiveModel
   autoload :Validator
 
   eager_autoload do
+    autoload :AttributeAssignmentError, "active_model/multiparameter_attribute_assignment"
```

Comment:
If `MultiparameterAttributeAssignment` is not inside `eager_autoload` (and it should not) we don't need to have the exceptions inside it since they are defined in the same file.